### PR TITLE
postgres integration with frontend 

### PR DESCRIPTION
**Pull request by yerx**
Screen Shot 2020-02-20 at 10 53 22 PM

- Data generation and upload to Postgres and Cassandra
- Connected server and frontend to fetch from Postgres database

### CODE REVIEW

File: database/cassandra/index.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  // },
+});
+
+const t0 = new Date();
+
+
+client.connect()
+  .then(() => {
+    console.log('start', new Date() - t0);
+    const query = 'SELECT sellerid FROM additionalitems WHERE itemid=28999999';
```
Comment (dmoygit) [inline database/cassandra/index.js:20]: is this a test query (hardcoded itemid)?

File: database/cassandra/index.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  // pooling: {
+  //   maxRequestsPerConnection: 20000,
+  // },
+});
+
+const t0 = new Date();
+
+
+client.connect()
+  .then(() => {
```
Comment (dmoygit) [inline database/cassandra/index.js:18]: every time you connect, it's going to run a query?

File: database/dataGen/genDataV02.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+        Object.entries(generateSellerInfo).forEach(([key, value]) => {
+          generatedItemInfo[key] = value;
+        });
+
+        const randomImgUrl = Math.floor(Math.random() * 50) + 1;
+
+        generatedItemInfo.itemId = k;
+        generatedItemInfo.itemName = faker.commerce.productName();
+        generatedItemInfo.itemPrice = (Math.random() * 1200).toFixed(2) + 5;
```
Comment (dmoygit) [inline database/dataGen/genDataV02.js:56]: If this is fixed to 2 decimal places, why are the prices in the image going to 3 decimal places?

File: database/postgres/index.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
+const { Pool } = require('pg');
```
Comment (dmoygit) [inline database/postgres/index.js:1]: should we use 'Client' now, to measure initial performance and then switch to 'Pool' as an optimization?

**Review by dmoygit — APPROVED** [review]
Approved so you can keep working.  I made some comments, but the only one that is consumer-facing is the price in the image (3 decimal places).  I'm not sure where that is happening since your faker data is fixed to 2 decimal places.  It might be somewhere else in your code that isn't in this PR.

File: database/cassandra/index.js (hunk 1 above)
Comment (yerx) [reply database/cassandra/index.js:20]: Yes, this was to run test queries.

File: database/cassandra/index.js (hunk 2 above)
Comment (yerx) [reply database/cassandra/index.js:18]: Yes, this script was for testing purposes.

File: database/dataGen/genDataV02.js (hunk 3 above)
Comment (yerx) [reply database/dataGen/genDataV02.js:56]: I don't know why that's happening. I checked my csv and it goes to 3 decimal places. I'll do a fix in the frontend to display up to 2 decimal places.

File: database/postgres/index.js (hunk 4 above)
Comment (yerx) [reply database/postgres/index.js:1]: Probably, I'll try both when measuring performance.

**Pull request merged**